vip/test/test_brito.py

```diff
@@ -0,0 +1,134 @@
+import sys
```

Comment:
Just checking-- you did want to keep this file as part of your PR? I remember it having some temporary purpose.

Reply:
Yeah, it is basically an old demo of the gradient flags. Removed.